kernel/packages/unity-interface/portableExperiencesUtils.ts

```diff
@@ -0,0 +1,138 @@
+import {
+  ContentMapping,
+  EnvironmentData,
+  LoadablePortableExperienceScene,
+  MappingsResponse,
+  SceneJsonData
+} from '../shared/types'
+import { getSceneNameFromJsonData } from '../shared/selectors'
+import { parseParcelPosition } from '../atomicHelpers/parcelScenePositions'
+import { UnityPortableExperienceScene } from './UnityParcelScene'
+import {
+  forceStopParcelSceneWorker,
+  getParcelSceneID,
+  getSceneWorkerBySceneID,
+  loadParcelScene
+} from 'shared/world/parcelSceneManager'
+import { unityInterface } from './UnityInterface'
+import { resolveUrlFromUrn } from '@dcl/urn-resolver'
+
+declare var window: any
+window['spawnPortableExperienceScene'] = spawnPortableExperienceScene
+window['killPortableExperienceScene'] = killPortableExperienceScene
+
+export type PortableExperienceHandle = {
+  pid: string
+  parentCid: string
+}
+
+let currentPortableExperiences: Map<string, string> = new Map()
+
+export async function spawnPortableExperienceScene(
+  sceneUrn: string,
+  parentCid: string
+): Promise<PortableExperienceHandle> {
+  const scene = new UnityPortableExperienceScene(await getPortableExperienceFromS3Bucket(sceneUrn))
```

Comment:
what would happen if I execute this call twice?

```ts
spawnPortableExperienceScene(urn, parentUrn)
spawnPortableExperienceScene(urn, parentUrn)
```

Will it create two copies of the same experience?

Reply:
No, it creates only one per id.